src/bin/server/worker.rs

```diff
@@ -307,7 +311,7 @@ impl Worker {
 ```
 
 </details>
-        "#, opening = opening, html_url = job.html_url(), log_url = log_url, log = extracted))?;
+{trailer}"#, opening = opening, html_url = job.html_url(), log_url = log_url, log = extracted, trailer = trailer))?;
```

Comment:
```suggestion
{trailer}"#, opening = opening, html_url = job.html_url(), log_url = log_url, log = extracted))?;
```
why is this required? does this project uses an old rust version?

Reply:
Not really, I just wanted to keep the original style.

Reply:
nostalgia, I get it 😛